internal/code-generator/generator/diff/diff.go

```diff
@@ -0,0 +1,58 @@
+package diff
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func writeTmpFile(dir, prefix string, data []byte) (string, error) {
```

Comment:
More readable imho
```suggestion
func writeTempFile(dir, prefix string, data []byte) (string, error) {
```

Reply:
Thank you. As applying this suggestion in github UI would result in compilation failure, I had to stash the ongoing work, change the branches and rename this private helper function 🙄  

History re-written.